BaseStation/PTZCamera/Camera_Control.cs

```diff
@@ -0,0 +1,81 @@
+﻿using OpenTK;
+using OpenTK.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HuskyRobotics
+{
+    /// <summary>
+    /// This class is just used to test the camera controls.
+    /// </summary>
+    class Camera_Control
+    {
+        static void Main(string[] args)
+        {
+            int max_speed = 100;
+            int x_speed=0;
+            int y_speed=0;
+            bool speed_Changed;
+            HttpClient client = new HttpClient();
+            PTZCamera cam = new PTZCamera("192.168.0.42", "admin","1234",client);
+            while (true)
+            {
+                speed_Changed = false;
+                if (Keyboard.GetState().IsKeyDown(Key.Up) )
+                {
+                    if(y_speed != max_speed){ 
+                        y_speed = max_speed;
+                        speed_Changed = true;
+                    }
+                }
+                else if (Keyboard.GetState().IsKeyDown(Key.Down))
+                {
+                    if (y_speed != -max_speed)
+                    {
+                        y_speed = -max_speed;
+                        speed_Changed = true;
```

Comment:
This speed_changed concept should be handled in the PTZ camera. It knows that if it successfully set the speed, then it does not need to set it again. This is an implementation detail in the connection for the PTZ camera, rather than for the keyboard control.

Right now, if I press a key and this fails to transmit, then pressing it again won't retransmit the message. This is a problem in a potentially unreliable connection environment. Moving the speed_changed concept into the PTZ camera fixes this.